caddytest/integration/caddyfile_adapt/not_block_with_named_matcher.txt

```diff
@@ -0,0 +1,107 @@
+:80
+
+@matcher {
+    header Abc "123"
+    header Bcd "456"
```

Comment:
Use tabs instead of spaces, avoids a `caddy fmt` warning.

```suggestion
	header Abc "123"
	header Bcd "456"
```

Reply:
Done.